.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 Cargo.lock
+*.swp
```

Comment:
`.swp` files are made by `ViM`, right?
I am not sure if including editor-specific files in the project's `.gitignore` is a good idea, since people may use any number of editors, (i.e. `emacs` creates `~` files and `IntelliJ` has `.idea` etc.), instead you may want to set up a [global `.gitignore`](http://stackoverflow.com/questions/7335420/global-git-ignore) for this.